numba/core/typing/templates.py

```diff
@@ -285,6 +285,28 @@ def get_impl_key(self, sig):
             key = key.im_func
         return key
 
+    @classmethod
+    def get_source_code_info(cls, impl):
+        """
+        Gets the source information about function impl.
+        Returns:
+
+        code - # str: source code as a string
+        firstlineno - # int: the first line number of the function impl
+        path - #str: the path to file containing impl
```

Comment:
the `#` looks odd here
```suggestion
        code - str: source code as a string
        firstlineno - int: the first line number of the function impl
        path - str: the path to file containing impl
```